src/deploy-cromwell-on-azure/scripts/helm/templates/persistence.yaml

```diff
@@ -21,8 +21,10 @@ metadata:
   name: {{ $namespace }}-blob-fuse
 provisioner: blob.csi.azure.com
 parameters:
-  skuName: Standard_LRS  # available values: Standard_LRS, Premium_LRS, Standard_GRS, Standard_RAGRS
+  skuName: Premium_LRS  # available values: Standard_LRS, Premium_LRS, Standard_GRS, Standard_RAGRS
```

Comment:
I think this should remain `Standard_LRS` since that's what we are using for TES, don't you think?

Reply:
That is one of the upgrade issues. You can't change any of the parameters of a storage class with a helm upgrade only.

Reply:
Actually, its occurs to me that this PR will fix the upgrade issues from 5.2.0 builds but will break upgrades for 5.3.0 -> 5.3.1 since we are changing the parameters again. We'll probably need to do it the hard way and delete all the storage classes and recreate then to universally fix it.

Reply:
Fixed by renaming it then helm will take care of the deletion and recreation on its own.